config/private.yml

```
@@ -1,5 +1,5 @@
 ---
 blobstore:
-  provider: s3
+  provider: sqs
```

```suggestion
  provider: s3
```

We cannot upload blobs to SQS